nginx/default.conf

```diff
@@ -0,0 +1,36 @@
+server {
```

Comment:
Perhaps can call this folder as `gateway` instead of generic `nginx`

Reply:
Sure